app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/common/OnTappedListener.java

```diff
@@ -0,0 +1,9 @@
+package com.xmartlabs.scasas.criminalintent.ui.common;
+
+/**
+ * Created by scasas on 2/13/17.
+ */
+public interface OnTappedListener<T> {
```

Comment:
Isn't better to just use an Action<T>? @matir91 what do you think, in accordance with his training?